components/AddRecommendations.js

```diff
@@ -85,12 +84,6 @@ const AddRecommendations = () => {
     );
   };
 
-  useEffect(() => {
-    fetchFoods().then(() => {
-      addRecommendation();
-    });
-  }, []);
-
   return (
     <Fragment>
```

Comment:
<> is an easier way to do this now

```suggestion
    <>
```

Reply:
Can't seem to suggest below but you get the idea